src/metabase/public_settings.clj

```diff
@@ -462,33 +434,79 @@
   :visibility :public
   :type       :string
   :audit      :getter
-  :enabled?   premium-features/enable-whitelabeling?
+  :feature    :whitelabeling
   :default    "app/assets/img/logo.svg")
 
 (defsetting application-favicon-url
   (deferred-tru "The url or image that you want to use as the favicon.")
   :visibility :public
   :type       :string
   :audit      :getter
-  :enabled?   premium-features/enable-whitelabeling?
+  :feature    :whitelabeling
   :default    "app/assets/img/favicon.ico")
 
 (defsetting show-metabot
   (deferred-tru "Enables Metabot character on the home page")
   :visibility :public
   :type       :boolean
   :audit      :getter
-  :enabled?   premium-features/enable-whitelabeling?
+  :feature    :whitelabeling
   :default    true)
 
 (defsetting show-lighthouse-illustration
   (deferred-tru "Display the lighthouse illustration on the home and login pages.")
   :visibility :public
   :type       :boolean
   :audit      :getter
-  :enabled?   premium-features/enable-whitelabeling?
+  :feature    :whitelabeling
   :default    true)
 
+(def ^:private help-link-options
+  #{:metabase :hidden :custom})
+
+(defsetting help-link
+  (deferred-tru
+   (str
+    "Keyword setting to control whitelabeling of the help link. Valid values are `:metabase`, `:hidden`, and "
+    "`:custom`. If `:custom` is set, the help link will use the URL specified in the `help-link-custom-destination`, "
+    "or be hidden if it is not set."))
+  :default    :default
+  :type       :keyword
+  :audit      :getter
+  :visibility :public
+  :feature    :whitelabeling
+  :default    :metabase
+  :setter     (fn [value]
+                (when-not (help-link-options value)
+                  (throw (ex-info (tru "Invalid help link option")
+                                  {:value value
+                                   :valid-options help-link-options})))
+                (setting/set-value-of-type! :keyword :help-link value)))
+
+(defn- validate-help-url
+  "Checks that the provided URL is either a valid HTTP/HTTPS URL or a `mailto:` link. Returns `nil` if the input is valid;
+  throws an exception if it is not."
+  [url]
+  (if-let [matches (re-matches #"^mailto:(.*)" url)]
+    (when-not (u/email? (second matches))
+      (throw (ex-info (tru "Invalid email address in mailto: link")
```

Comment:
```suggestion
      (throw (ex-info (tru "Please make sure this is a valid URL")
```

Reply:
BTW: this change will make one of the e2e fail, I'll update the e2e to expect the correct message